### ISSUE TITLE
Clarify OMIM Identifier

### ISSUE THREAD

**Issue by sgoehringer** [clingen-data-model/data-exchange-shared-json#22]
@ronakypatel would it be possible to clarify the format of the OMIM identifier when you have a chance.

**Comment by ronakypatel** [comment]
Does this sound good?

```
        "conditions": 
        [
            {
                "ontology": "OMIM",
                "curie": "OMIM:114480",
                "uri": "OMIM114480",
                "iri": "http://purl.obolibrary.org/obo/OMIM_114480"
            }
       ]
```

**Comment by sgoehringer** [comment]
Great!

**Comment by sgoehringer** [comment]
Addressed - 53557c25bdccc5a0bc947c6104df14f3dfa0903e
